### ISSUE TITLE
Main branch is not default

### ISSUE THREAD

**Issue by zakkg3** [netbox-community/netbox-chart#175]
dismiss this and read the next comment.
```
Values files specify the need for keys on the existingSecret, but "napalm_password" is not there and the pod does not start if this is missing.

  Warning  FailedMount             85s (x2 over 3m27s)   kubelet                  MountVolume.SetUp failed for volume "secrets" : references non-existent secret key: napalm_password

```

**Comment by zakkg3** [comment]
I found the problem is that the default branch here is not main and you land by default on dev branch so the 5.0.0 chart which is not published. 

The latest we can use is 4.1.1 and this is on main branch, wich is not the main one.  o,0

very confusing, And I am pretty sure I am not the first one misled by this.

I guess the most sane would be to set main as the default branch and all this will be solved, like in every single reasonable repo

**Comment by tpzumezawa** [comment]
Happened to us :(

**Comment by bootc** [comment]
Yeah I could do that, but it's a double-edged sword. People contributing PRs would target the wrong branch and end up with wasted work/time fixing things that are already fixed, or having to redo their changes because what they're targeting has changed. GitHub unfortunately makes this very difficult.

**Comment by LeoColomb** [comment]
Based on the new release workflow, I think it is fine to drop the `master` branch and rename `develop` to `main`.
The complete history is still part of the main branch, and the tags keep the stable git HEAD available.
What do you think @netbox-community/netbox-chart-maintainers?

**Comment by RangerRick** [comment]
> Based on the new release workflow, I think it is fine to drop the `master` branch and rename `develop` to `main`. The complete history is still part of the main branch, and the tags keep the stable git HEAD available. What do you think @netbox-community/netbox-chart-maintainers?

Yeah, makes sense to just use once since we've got a just-in-time thing going now. That said, maybe it makes sense to merge `develop` over to `main`, and get rid of `develop`? It would signal better how the current workflow is, IMHO.

**Comment by LeoColomb** [comment]
I believe that is the same, with a merge step in addition 😅 (there is no `main` at the moment)

**Comment by RangerRick** [comment]
Oops I meant `master` in this case, not `main`.

**Comment by RangerRick** [comment]
Since we've moved to a more dynamic workflow in general (single branch for release -- currently `develop` -- and then everything comes from PRs) I will fix this when 5.0.0 rolls out for real.

**Comment by RangerRick** [comment]
FYI, 5.0.0 is merged to `develop`, and GitHub has been updated to treat `develop` as the main branch. The old `master` branch has been archived. Marking as closed.
